### PR TITLE
Add tests for bad commands for no args and 1 arg(int)

### PR DESCRIPTION
**Pull request by theDrsh**

### CODE REVIEW

File: mako_files/ucoms_decode.cc.mako [hunk 1]
```diff
@@ -16,7 +16,7 @@ char uComsDecode::Increment(int* index, const char* input) {
 int uComsDecode::ParseValue(int index, const char* input, const char end_char) {
```
Comment (theDrsh) [inline mako_files/ucoms_decode.cc.mako:16]: Maybe should rename this since it's not just parsing value, it's parsing the substring.

File: tests/ucoms_decode_test.cc [hunk 2]
```diff
@@ -44,6 +44,60 @@ TEST_F(test_uComsDecode, decoderTest) {
   }
 }
 
+TEST_F(test_uComsDecode, noArgBadCmd) {
```
Comment (theDrsh) [inline tests/ucoms_decode_test.cc:47]: I'm sure there are a lot more edge cases but this is a good start.
